src/inspector/utils.zig

```diff
@@ -0,0 +1,3 @@
+pub fn toKiloBytes(bytes: usize) usize {
```

Comment:
This is kibibytes not kilobytes, and as a result should be displayed as KiB not KB.

Reply:
@JakeWharton Thanks for the heads up on this one. Since this is representing memory size I think kibibyte is the most appropriate.